backend/src/users/users.service.ts

```diff
@@ -114,4 +128,20 @@ export class UsersService {
       },
     });
   }
+
+  verifyMagicNum(filePath: string): boolean {
+    const data: string = fs
+      .readFileSync(filePath, { encoding: 'hex' })
+      .slice(0, 8)
+      .toUpperCase();
+
+    if (
+      data == hexSignature.GIF ||
+      data == hexSignature.PNG ||
+      data.slice(0, 6) == hexSignature.JPG
+    )
+      return true;
+    this.remove_avatar(filePath, ' : invalid signature.');
```

Comment:
```suggestion
    this.remove_avatar(filePath, ': invalid signature.');
```
There is no space in English before a colon.